### PR TITLE
Add FamilyUtils extensions

### PR DESCRIPTION
**Pull request by SergeyNefyodov**
# Summary of the Pull Request

I added support for 2 methods of FamilyUtils class

**Description:** 

## Quality Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: source/RevitLookup/Core/ComponentModel/Descriptors/FamilyDescriptor.cs [hunk 1]
```diff
@@ -33,5 +34,22 @@ public override Func<IVariants> Resolve(Document context, string target, Paramet
...
     {
         manager.Register(nameof(FamilySizeTableManager.GetFamilySizeTableManager), context => FamilySizeTableManager.GetFamilySizeTableManager(context, family.Id));
+        manager.Register(nameof(FamilyUtils.FamilyCanConvertToFaceHostBased), context => FamilyUtils.FamilyCanConvertToFaceHostBased(context, family.Id));
+        manager.Register(nameof(FamilyUtils.GetProfileSymbols), ResolveProfileSymbols);
+    }
+    
+    private static object ResolveProfileSymbols(Document context)
+    {
+        var values = Enum.GetValues(typeof(ProfileFamilyUsage));
+        var capacity = values.Length;
```
Comment (Nice3point) [inline source/RevitLookup/Core/ComponentModel/Descriptors/FamilyDescriptor.cs:44]: Multiply by 2. You have two results for one value

File: source/RevitLookup/Core/ComponentModel/Descriptors/FamilyDescriptor.cs [hunk 2]
```diff
@@ -33,5 +34,22 @@ public override Func<IVariants> Resolve(Document context, string target, Paramet
...
+        var variants = new Variants<ICollection<ElementId>>(capacity);
+        
+        foreach (ProfileFamilyUsage value in values)
+        {
+            variants.Add(FamilyUtils.GetProfileSymbols(context, value, false), value.ToString());
+            variants.Add(FamilyUtils.GetProfileSymbols(context, value, true), $"{value.ToString()}, One curve loop only");
```
Comment (Nice3point) [inline source/RevitLookup/Core/ComponentModel/Descriptors/FamilyDescriptor.cs:50]: Redundant .ToString() inside interpolation

File: source/RevitLookup/Core/ComponentModel/Descriptors/FamilyDescriptor.cs [hunk 3]
```diff
@@ -33,5 +34,22 @@ public override Func<IVariants> Resolve(Document context, string target, Paramet
...
+    
+    private static object ResolveProfileSymbols(Document context)
+    {
+        var values = Enum.GetValues(typeof(ProfileFamilyUsage));
+        var capacity = values.Length;
+        var variants = new Variants<ICollection<ElementId>>(capacity);
+        
+        foreach (ProfileFamilyUsage value in values)
+        {
+            variants.Add(FamilyUtils.GetProfileSymbols(context, value, false), value.ToString());
```
Comment (Nice3point) [inline source/RevitLookup/Core/ComponentModel/Descriptors/FamilyDescriptor.cs:49]: We have to add extra description to understand context. For example, this method with $"{value} with multiple curve loops" and next $"{value} with single curve loop"

**Review by Nice3point — CHANGES_REQUESTED** [review]

File: source/RevitLookup/Core/ComponentModel/Descriptors/FamilyDescriptor.cs (hunk 3 above)
Comment (SergeyNefyodov) [reply source/RevitLookup/Core/ComponentModel/Descriptors/FamilyDescriptor.cs]: Forgot about this PR. I fixed it all

**Pull request merged**